R/style_rmd.R

```diff
@@ -0,0 +1,67 @@
+#' Transform code from R or Rmd files
+#'
+#' @inheritParams utf8::transform_lines_enc
+#' @param ... Further arguments passed to `utf8::transform_lines_enc()`.
+transform_code <- function(path, fun, verbose, ...) {
+  if (grepl("\\.R$", path, ignore.case = TRUE)) {
+    utf8::transform_lines_enc(path, fun = fun, ...)
+  } else if (grepl("\\.Rmd$", path, ignore.case = TRUE)) {
+    utf8::transform_lines_enc(path, fun = partial(transform_rmd, transformer_fun = fun), ...)
+  } else {
+    stop("")
+  }
+}
+
+#' Transform Rmd contents with a transformer function
+#'
+#' @param lines A character vector of lines from an Rmd file
```

Comment:
Can you use tidyverse style in your roxygen comments? In particular, argument descriptions should end with a full stop here and in other places.